src/docs/_layouts/items.html

```diff
@@ -19,6 +19,38 @@ <h2 class="article-header" id="{{element.name | slugify}}">
       {% include {{elementPath}} %}
     </article>
     {% endfor %}
+
+  <pre class="copy-helper"><code class="copy-helper__code"></code></pre>
+  <script>
+    var copyHelper = document.querySelector('.copy-helper__code');
+
+    document.body.addEventListener('click', function(e) {
+      var prevTarget = null;
+      var target = e.target;
+
+      while(target.tagName.toLowerCase() === 'use' || target.classList.toString().indexOf('sg-') !== -1) {
```

Comment:
so... this is not as simple as 'get code of clicked element'. I'm looking for the highest/oldest parent with `sg-` in the classList. This selects e.g. whole `sg-box` instead of `sg-box__hole`.